lms/static/sass/views/_homepage.scss

```diff
@@ -32,6 +32,7 @@ $learn-more-horizontal-position: calc(50% - 100px); // calculate the left positi
 
       .course-image .cover-image {
         height: $course-image-height;
+        background: $uxpl-blue-base;
```

Comment:
@marcotuts Another idea I had though, was  to make the button blue, looking like our other buttons. If we did that, then the background could be white, or even left unset like it was previously.